dev/build-ballista-docker.sh

```diff
@@ -19,9 +19,7 @@
 
 set -e
 
-docker build -t ballista-builder --build-arg EXT_UID="$(id -u)" -f dev/docker/ballista-builder.Dockerfile .
```

Comment:
This shouldn't be in here - I must have had git problems... please don't merge until I can review.